code/_onclick/cyborg.dm

```diff
@@ -95,14 +95,12 @@
 		return
 
 	// cyborgs are prohibited from using storage items so we can I think safely remove (A.loc && isturf(A.loc.loc))
-	if(isturf(A) || isturf(A.loc))
-		if(A.Adjacent(src)) // see adjacent.dm
-			W.melee_attack_chain(src, A, params)
-			return
-		else
-			W.afterattack(A, src, 0, params)
-			return
-	return
+	if(can_reach(A,W))
+		W.melee_attack_chain(src, A, params)
+		return
+	else
```

Comment:
no need for this else, above if returns always

Reply:
done